ui/selectors/selectors.js

```diff
@@ -927,6 +927,23 @@ export const getMemoizedTargetSubjectMetadata = createDeepEqualSelector(
   (interfaces) => interfaces,
 );
 
+/**
+ * Get a memoized version of the unapproved confirmations.
+ */
+export const getMemoizedUnapprovedConfirmations = createDeepEqualSelector(
```

Comment:
If we're adding new selectors, would a better location be `selectors/approvals.ts` so we don't bloat this file plus can use TypeScript and keep them similar domain grouped together?

Reply:
If you want TypeScript support for these selectors, we would most likely need to move a bunch more things to TypeScript. I would argue out of scope for this PR.